.drone.jsonnet

```diff
@@ -0,0 +1,24 @@
+local Pipeline(os, arch, version) = {
+    kind: "pipeline",
+    name: os+" - "+arch+" - Julia "+version,
+    platform: {
+	os: os,
+	arch: arch
+    },
+    steps: [
+	{
+	    name: "build",
+	    image: "julia:"+version,
+	    commands: [
+		"julia --project=. --check-bounds=yes --color=yes -e 'using InteractiveUtils; versioninfo(verbose=true); using Pkg; Pkg.build(); Pkg.test(coverage=true)'"
+	    ]
+	}
+    ]
+};
+
+[
+    # Pipeline("linux", "arm",   "1.0"),
```

Comment:
@giordano, why was this commented out? Is this something we could try enabling?

Reply:
When this is fixed: https://github.com/JuliaMath/FFTW.jl/issues/104 🙂

Reply:
#184 :slightly_smiling_face: